listenbrainz/webserver/static/js/src/user-feed/UserFeed.tsx

```diff
@@ -607,6 +607,26 @@ export default class UserFeedPage extends React.Component<
 
                           <span className="event-time">
                             {preciseTimestamp(created * 1000)}
+                            {((event.event_type ===
+                              EventType.RECORDING_RECOMMENDATION ||
+                              event.event_type === EventType.RECORDING_PIN) &&
+                              event.user_name === currentUser.name) ||
+                            event.event_type === EventType.NOTIFICATION ? (
+                              <ListenControl
+                                title=""
+                                icon={faTrash}
+                                buttonClassName="btn btn-link"
+                                // eslint-disable-next-line react/jsx-no-bind
+                                action={this.deleteFeedEvent.bind(this, event)}
+                              />
+                            ) : (
+                              <ListenControl
```

Comment:
So after seeing the page with all the garbage cans, I'm starting to have misgivings about the solution.

I propose one of two things:

1. Add the class `.btn-xs` to make all the buttons much smaller, therefore taking less space and being less conspicuous visually:
![image](https://user-images.githubusercontent.com/6179856/154737164-ae2ae395-1137-46a8-8b88-2f1d334b89f9.png)


2. Put the trash icon somewhere else and only on events the user can delete.
The first thing that came to mind to me was simply after the event description:
![image](https://user-images.githubusercontent.com/6179856/154737065-867702cb-5983-4ede-ad39-55a13912a91f.png)

We could also make it appear on the left-hand side, maybe when we hover over the event for example?
![image](https://user-images.githubusercontent.com/6179856/154737958-2875b031-0d4c-4864-941e-9ea62d894be1.png)



Sorry about the back and forth 😅 
What are your thoughts on this?

Reply:
Follow up of this conversation in IRC, for posterity:
https://chatlogs.metabrainz.org/libera/metabrainz/2022-02-22/?msg=4957942&page=1